collection.yaml

```diff
@@ -35,4 +36,26 @@ config:  # todo: clean up config (which of these fields are still used?)
   default_type: model
   url_root: https://raw.githubusercontent.com/bioimage-io/bioimage-io-resources/main
   
-collection: []
+collection:
+- id: stardist
+  name: StarDist
+  description: "Object Detection with Star-convex Shapes"
+  format_version: 0.2.2
+  source: "https://github.com/stardist/stardist"
```

Comment:
Don't we need source to be the link to an actual bioengine app?

Reply:
It doesn't need to be an runnable bioengine app, you can define it as an regular application (you need type: application), I would skip the source field, set github_repo instead. And it's better to add other field  ssuch as documentation, authors, tags.